shellpromise.js

```diff
@@ -32,7 +32,7 @@ module.exports = function(processToRun, options) {
 				resolve(output);
 			} else {
 				if (options.verbose) {
-					toStdErr(processToRun + ' exited with exit code ' + code);
+					console.warn(processToRun + ' exited with exit code ' + code);
```

Comment:
@acolby i've switched this to a `console.warn` because i didn't like `childpromise` putting anything in the `output` that wasn't output by the command actually being executed (`toStdErr` prints to the console *AND* puts stuff in the `output` variable).